exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -192,6 +193,19 @@ public Builder setChannel(ManagedChannel channel) {
       return this;
     }
 
+    /**
+     * Sets the jaeger endpoint. This will use the endpoint to assign a {@link ManagedChannel}
+     * instance to this builder.
+     *
+     * @param endpoint The Jaeger endpoint URL, ex. "jaegerhost:14250".
+     * @return this.
+     * @since 0.7.0
+     */
+    public Builder setEndpoint(String endpoint) {
+      setChannel(createChannel(endpoint));
```

Comment:
I would not create the channel here. I would set the default endpoint in the ctor of the builder. Then in `build()` will check if the channel was set externally otherwise would create a new channel from "endpoint".

It feels to my more natural.

Reply:
Makes sense, done